src/util/StreamUtils.h

```diff
@@ -0,0 +1,110 @@
+//   Copyright 2023, University of Freiburg,
+//   Chair of Algorithms and Data Structures.
+//   Author: Robin Textor-Falconi <textorr@informatik.uni-freiburg.de>
```

Comment:
This file is not `StreamUtils` it is `DurationParsers` or `ParseableDuration` or something like that.